src/Evolution/Executables/GeneralizedHarmonic/CMakeLists.txt

```diff
@@ -12,16 +12,16 @@ function(add_generalized_harmonic_executable_without_horizon
     )
 endfunction(add_generalized_harmonic_executable_without_horizon)
 
-function(add_generalized_harmonic_cce_executable_without_horizon
-  EXECUTABLE_NAME INITIAL_DATA LIBS_TO_LINK)
+function(add_generalized_harmonic_ccm_executable_without_horizon
+  EXECUTABLE_NAME INITIAL_DATA EVOLVE_CCM LIBS_TO_LINK)
   add_spectre_parallel_executable(
-    "EvolveGhCce${EXECUTABLE_NAME}"
-    EvolveGeneralizedHarmonicCce
+    "EvolveGh${EXECUTABLE_NAME}"
+    EvolveGeneralizedHarmonicCcm
```

Comment:
These names are ok though because they are the actual executables and we want those to be Cce and Ccm